website/content/docs/enterprise/network-segments.mdx

```diff
@@ -11,19 +11,200 @@ description: |-
+   the agent (e.g., `consul agent -retry-join "client-b-address:8303"`), or in the agent's
+   configuration using the [retry-join option](/docs/agent/options#retry_join). This method
+   is not compatible with [cloud auto-join](/docs/install/cloud-auto-join#auto-join-with-network-segments).
+
+2. **Specify an alternate Serf LAN port for the agent**. This can be done at the CLI when starting
+   the agent (e.g., `consul agent -retry-join "client-b-address" -serf-lan-port 8303`), or in
+   the agent's configuration using the [serf_lan](/docs/agent/options#serf_lan_port) option.
+   When a Serf LAN port is not explicitly specified in the join address, the agent will use its
+   own Serf LAN port number as the join target port. This method requires that the agent and its
+   join target use the same Serf LAN port for this segment.
+
+3. **Use the default Serf LAN port (8301)**. This method requires that the agent and its join target use
+   the same Serf LAN port for this segment.
+
+-> Agents within a segment can use different port numbers for their Serf LAN port.
+For example, on the `<default>` segment, Server S can use port 8301, Client A
+can use 8303, and Client B can use 8304. However, if an agent wishes to join a
+segment via an agent that uses a different port number, the target agent's Serf LAN
+port must be specified in the join address (method 1 above).
+
+### Example Client Configuration
+
+The following client agent configuration will cause the agent to:
+
+- Open a Serf LAN listener port on 8303.
+- Attempt to join the cluster via servers on port 8303 (since an alternate port is not
+  specified in the `retry_join` addresses).
+
+<CodeTabs heading="Example network segment configuration for client agents">
+
+```hcl
+node_name = "consul-client1"
+retry_join = ["consul-server1", "consul-server2", "consul-server3"]
+segment = "alpha"
+ports = {
+  serf_lan = 8303
+}
+```
+
+```json
+{
+  "node_name": "consul-client1",
+  "retry_join": ["consul-server1", "consul-server2", "consul-server3"],
+  "segment": "alpha",
+  "ports": {
+    "serf_lan": 8303
+  }
+}
+```
+
+</CodeTabs>
+
+The client [agent configuration](/docs/agent/options) options relevant to network
+segments are:
+
+- [`segment`](/docs/agent/options#segment-2): The name of the network segment this
+  client agent belongs to.
+- [`ports.serf_lan`](/docs/agent/options#serf_lan_port):
+  Serf LAN port for the above segment on this client. This is not required
+  to match the configured Serf LAN port for other agents on this segment.
+- [`retry_join`](/docs/agent/options#retry_join) or
+  [`start_join`](/docs/agent/options#start_join): A list of agent addresses to join
+  when starting. Ensure the correct Serf LAN port for this segment is used when joining
+  the LAN gossip pool using one of the [available configuration methods](#join_a_client_to_a_segment).
 
 # Consul Networking Models
```

Comment:
@blake : I _think_ the last remaining question I have is where this Core Networking Models section belongs.

You previously said:
> The tutorial on the network segments page should probably come after the Consul Networking Models section so the user has the proper context on the networking models before reading how to configure the feature.

> Another option would be to move the tutorial be moved to a different page entirely and referenced from this main page. Although, at that point it may duplicate what is on the Learn guide. 🤔

I view the Learn guide as a "How To" whereas this is page is more of a complete reference. It could possibly make sense to separate a Network Segments overview from the details, though I don't think this page is soooo long as to make that necessary. I'm thinking we leave it all on this one page for now.

I think the first section on this page should definitely be the Network Segments overview that is shown right now (with the images), not the Core Networking Models. 

Perhaps it could look like the below. Do you agree?
```
# Network Segments
// ... the overview explanation with the images ...

# Consul Networking Models
// ...

# Configuring Network Segments

## Server Configuration
### Example Server Configuration
// ... the rest of the new content
```

Reply:
@blake : I'll go through your last round of comments (from Aug 23) soon, but before I do, I want to see if you have an opinion on this (where the Core Networking Models section belongs). I'd like to batch all the remaining changes into one edit if possible.